scripts/upgrades/Dockerfile.cosmovisor

```diff
@@ -2,6 +2,7 @@
 FROM golang:1.18-alpine3.15 AS builder
 
 ARG old_commit_hash
+ARG STRIDE_ADMIN_ADDRESS
```

Comment:
nit: can you either set these both to uppercase or both to lowercase to keep the styling consistent?